hack/jenkins/e2e.sh

```diff
@@ -106,6 +106,22 @@ Addon\
 |multiport\sendpoints\
 "
 
+# release-0.21 GCE skip list
+RELEASE_0_21_GCE_SKIP_TEST_REGEX="\
```

Comment:
It's not clear to me how these are any different than GCE_SKIP_TEST_REGEX at the appropriate release?